docs/v1.1.0/schema.md

```diff
@@ -0,0 +1,216 @@
+|                               |                              | Email                       | email                       | STRING                     |  --   |   --    | OPTIONAL  for non-corresponding author; MUST for corresponding author | N/A                                                            | Email address for each annotator                                                                                                                                                                                   |
+|                               |                              | Affiliation name            | affiliation_name            | STRING                     |  --   |   --    | RECOMMENDED                                                           | ROR                                                            | Name of the institution an annotator is affiliated with. Sometimes, one annotator may have multiple affiliations.                                                                                                  |
+|                               |                              | Affiliation address         | affiliation_address         | STRING                     |  --   |   --    | OPTIONAL                                                              | N/A                                                            | Address of the institution an annotator is affiliated with.                                                                                                                                                        |
+|                               |                              | Affiliation identifier      | affiliation_identifier      | STRING                     |  --   |   --    | RECOMMENDED                                                           | ROR                                                            | A unique identifier assigned to the affiliated institution by The Research Organization Registry (ROR).                                                                                                            |
+|                               |                              | Author list order           | author_list_order           | INTEGER                    |  --   |   --    | MUST                                                                  |                                                                | The order in which the author appears in the publication                                                                                                                                                           |
```

Comment:
The entire Author section is actually not a dictionary, but a list of Dictionaries of type "TomogramAuthor". We should reflect this in the documentation.